src/services/learninghobbies/index.jsx

```
@@ -249,6 +251,289 @@ export const AddLearningHobbies = ({
+          video: Yup.string().optional(),
+          thumbnail: Yup.string().optional(),
+        })}
+        onSubmit={handleEditLearningHobbies}
+      >
+        {({ handleSubmit, values, handleChange, setFieldValue }) => (
+          <div className="flex-col-start gap-5">
+            <div className="w-full flex items-center justify-center gap-5">
+              <div
+                className={`relative capture-container`}
+                onClick={() => videoRef.current.click()}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)